solidus_paypal_braintree.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
   end
 
   s.add_dependency 'activemerchant', '~> 1.48'
-  s.add_dependency 'braintree', '~> 3.4'
+  s.add_dependency 'braintree', '~> 4.0.0'
```

Comment:
let's do '~> 4.0' please, so we can use 4.20 for example,that version worked fine here https://github.com/chordcommerce/chord-oms/pull/2325/commits/94731fc9228e3cf3b58d3701a82ea931f53b4538#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR279

Reply:
Ok done, should be good now.